css-canvas-medical/styles.css

```diff
@@ -0,0 +1,298 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Tiempos Headline', serif;
+  font-style: normal;
+  font-weight: normal;
+  src: url('TiemposHeadline-Regular.woff2') format('woff2');
+  color: rgb(99, 99, 99);
+}
+
+p,
+a {
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url('avenir.woff2') format('woff2');
+  color: rgb(131, 131, 131);
+}
+
+.container {
+  padding-left: 2rem;
+  padding-right: 2rem;
+}
+
+.row {
+  display: flex;
+  padding: 1.2rem 1rem;
+}
+
+.one-fifth-column {
+  display: flex;
+  flex-basis: 20%;
+}
+
+.four-fifth-column {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  flex-basis: 80%;
+}
+
+.four-fifth-column a {
+  padding: 0 1.5rem;
+}
+
+.blue-logo {
+  width: 9rem;
+  height: 2.6rem;
+}
+
+.contact-us-button {
+  background-color: rgb(41, 68, 100);
+  color: white;
+  padding: 8px 20px;
+  border-radius: 3px;
+  border-style: unset;
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url('avenir.woff2') format('woff2');
+  font-size: 16px;
+}
+
+.row2 {
+  padding: 3rem 0;
+}
```

Comment:
This is tough. There is a thing call a "hero" section. I'd use that for these areas with text over a background image. `row1`, `row2`, `row3` is not a good habit to get into. If you are having a hard time naming a section of the page, feel free to steal some of the content.

```css
.hero {

}

.hero-column {

}

.clinicians-prefer {

}
```

We want to get away from just numbering out classes, but naming is hard so we gotta get kind of creative.